gcloud/storage/test_api.py

```diff
@@ -58,7 +58,7 @@ def _lookup_bucket_hit_helper(self, use_default=False):
         ])
         http = conn._http = Http(
             {'status': '200', 'content-type': 'application/json'},
-            '{"name": "%s"}' % BLOB_NAME,
+            '{{"name": "{0}"}}'.format(BLOB_NAME).encode(),
```

Comment:
Please make the encoding explicit (otherwise, tests mail fail on machines w/ different defaults).